Runbooks/startstop/1.0/AutoStop_CreateAlert_Child.ps1

```diff
@@ -0,0 +1,321 @@
+﻿<#
+.SYNOPSIS
+ Child runbook for AutoStop scenario to create alerts (both classic and V2 ARM alerts) for the given VMs
+.DESCRIPTION
+ Child runbook for AutoStop scenario to create alerts (both classic and V2 ARM alerts) for the given VMs
+.EXAMPLE
+.\AutoStop_CreateAlert_Child.ps1 -VMObject "VM" -AlertAction "Create" -WebhookUri "url"
+Version History
```

Comment:
Could consider putting the version history in a .NOTES field

Reply:
Microsoft created this script. We can optimize and fix them in a different PR.